Docs layout is broken around 1280px

Hi there, I am not sure if this is the right repo to create an issue into but the layout on [reacttraining.com](https://reacttraining.com/react-router/) breaks here and there when the browser width is around 1280px.

Example:
![screen shot 2018-03-30 at 2 42 54 pm](https://user-images.githubusercontent.com/3950855/38149530-d674ecd6-3428-11e8-8259-bcf1456f316f.png)

Maybe the error messages will help 😀

```
ERROR in ./modules/components/App.js
Module not found: Error: Can't resolve '../../../packages/react-router-dom/BrowserRouter' in '/Users/vgrafe/Perso/react-router/website/modules/components'
 @ ./modules/components/App.js 9:21-80
 @ ./modules/index.js
 @ multi ../node_modules/webpack-dev-server/client?http://0.0.0.0:8081 ./modules/index.js

ERROR in ./modules/ReactRouterDOMShim.js
Module not found: Error: Can't resolve '../../packages/react-router-dom' in '/Users/vgrafe/Perso/react-router/website/modules'
 @ ./modules/ReactRouterDOMShim.js 6:22-64
 @ ./modules/components/App.js
 @ ./modules/index.js
 @ multi ../node_modules/webpack-dev-server/client?http://0.0.0.0:8081 ./modules/index.js
```

--- a/website/modules/base.css
+++ b/website/modules/base.css
@@ -131,7 +131,7 @@ markdown h1, h2, h3 {
   clear: both;
 }
 
-@media screen and (max-width: 1170px) {
+@media screen and (max-width: 1270px) {
   .api-doc-wrapper {
     background: white;
   }

